### PR TITLE
Optional semi

### LINKED ISSUE #2801: Conversion scripts not defined
Conversion scripts as such are not defined in the specification. In chapter 18.8.2 it is only mentioned that there may be a reference (script="convertTo3.mos") to such a script and that there are some functions which perform some operation.
But what is allowed inside such a script and especially whether the calls to these functions are separated by a semicolon or just by spaces is not specified (there are no semicolons in the current MSL 4.0 conversion scripts).  Someone might get the idea that the syntax is the same as in algorithm sections, but it is not. 

In the listing of the conversion functions they are mostly define without trailing semicolon, but the two possible calls of convertMessage have a trailing semicolon. That is odd.

### PR DESCRIPTION
**Pull request by HansOlsson**
Closes #2801

### CODE REVIEW

File: chapters/annotations.tex [hunk 1]
```diff
@@ -1813,25 +1814,25 @@ \subsubsection{Conversion rules}\label{conversion-rules}
 convertClass("Modelica.Thermal.FluidHeatFlow.Components.IsolatedPipe",
              "Modelica.Thermal.FluidHeatFlow.Components.Pipe")
```
Comment (eshmoylova) [inline chapters/annotations.tex:1815]: One more case here (unless we want to show that they are optional)
```suggestion
             "Modelica.Thermal.FluidHeatFlow.Components.Pipe");
```

File: chapters/annotations.tex [hunk 2]
```diff
@@ -1813,25 +1814,25 @@ \subsubsection{Conversion rules}\label{conversion-rules}
...
 convertModifiers({"Modelica.Thermal.FluidHeatFlow.Components.IsolatedPipe"},
-   fill("",0), {"useHeatPort=false"})
+   fill("",0), {"useHeatPort=false"});
 
 convertClass("Modelica.StateGraph.Temporary.NumericValue",
              "Modelica.Blocks.Interaction.Show.RealValue")
```
Comment (eshmoylova) [inline chapters/annotations.tex:1820]: Here as well.
```suggestion
             "Modelica.Blocks.Interaction.Show.RealValue");
```

File: chapters/annotations.tex [hunk 3]
```diff
@@ -1813,25 +1814,25 @@ \subsubsection{Conversion rules}\label{conversion-rules}
...
-   {"hideConnector"}, {"use_numberPort=not %hideConnector%"})
+   {"hideConnector"}, {"use_numberPort=not %hideConnector%"});
 
 convertModifiers("Modelica.Blocks.Math.LinearDependency",
    {"y0=0", "k1=0", "k2=0"}, {"y0=%y0%", "k1=%y0%*%k1%", "k2=%y0%*%k2%"},
-   true)
+   true);
 convertClass(
    "Modelica.Electrical.Machines.BasicMachines.QuasiStationaryDCMachines",
    "Modelica.Electrical.Machines.BasicMachines.QuasiStaticDCMachines")
```
Comment (eshmoylova) [inline chapters/annotations.tex:1831]: And here.
```suggestion
   "Modelica.Electrical.Machines.BasicMachines.QuasiStaticDCMachines");
```

**Review by eshmoylova — CHANGES_REQUESTED** [review]
Should there also be semicolon in the paragraphs's headings. For example, on line 1691 (unmodified version):
```
\paragraph*{convertClass("OldClass", "NewClass")}\label{convertclassoldclassnewclass}\annotationindex{convertClass}
```
Should it be:
```
\paragraph*{convertClass("OldClass", "NewClass");}\label{convertclassoldclassnewclass}\annotationindex{convertClass}
```
(Since these lines have not changed it does not allow me to comment inline.)
There are semi-colons in the convertMessage paragraphs on lines 1870 and 1880 (unmodified version).

**Comment by HansOlsson** [comment]
> Should there also be semicolon in the paragraphs's headings. 

My preference is for not having them, similarly as you don't use a period at the end of a heading.

> There are semi-colons in the convertMessage paragraphs on lines 1870 and 1880 (unmodified version).

Ok, they should be changed then to be consistent.

**Review by gkurzbach — APPROVED** [review]
With the changes requested by Elena I agree to the changes.

**Comment by eshmoylova** [comment]
> > Should there also be semicolon in the paragraphs's headings.
> 
> My preference is for not having them, similarly as you don't use a period at the end of a heading.
> 
> > There are semi-colons in the convertMessage paragraphs on lines 1870 and 1880 (unmodified version).
> 
> Ok, they should then be changed to be consistent.

I would be OK with removing them from convertMessage.

**Comment by HansOlsson** [comment]
> > > Should there also be semicolon in the paragraphs's headings.
> > 
> > 
> > My preference is for not having them, similarly as you don't use a period at the end of a heading.
> > > There are semi-colons in the convertMessage paragraphs on lines 1870 and 1880 (unmodified version).
> > 
> > 
> > Ok, they should then be changed to be consistent.
> 
> I would be OK with removing them from convertMessage.

That is now done.

**Review by eshmoylova — APPROVED** [review]
Looks good.

**Pull request merged**